KPCTabsControl/ChromeStyle.swift

```diff
@@ -137,47 +137,49 @@ public struct ChromeStyle: Style {
         let height: CGFloat = PaddedHeight.fromFrame(frame).value
         let xOffset = height / 2.0
         
-        let lowerLeft  = frame.origin + Offset(x: 0, y: frame.height)
+        let lowerLeft  = frame.origin + Offset(y: frame.height)
         let upperLeft  = lowerLeft + Offset(x: xOffset, y: -height)
-        let lowerRight = lowerLeft + Offset(x: frame.width, y: 0)
+        let lowerRight = lowerLeft + Offset(x: frame.width)
         let upperRight = lowerRight + Offset(x: -xOffset, y: -height)
 
+        let curve = CGFloat(3)
+
         let path = NSBezierPath()
 
         // TODO: Remove these awful hardcoded values...
-        // Remember that Origin start in upper left corner.
+        // Remember that Origin start in lower left corner.
```

Comment:
I think this was source of an error in the arc at the bottom where you ended up with `y-5=(-5)`. I found 5pt to be a bit too much, at least for smaller sizes, but we can play with this some more and make the value depend on the height of the tabs, for example.